package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "remark-preset-github",
   "description": "GitHub markdown and prose style",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
Hey! It's cool that you bumped this here since it's your first time contributing -- usually package authors prefer pull requests to not alter the version number.  No worries here but in the future if you just leave it up to the package author that should do the trick! Thanks again!

Reply:
Hey! I'm really sorry. I wasn't actually sure whether to do that; I guess I was enthusiastic! ☺️ Thank you for pointing it out, though. Still learning my contribution etiquette. 😄

_Sent from my Pixel 2 XL using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
Thanks again and great contribution!!!!!!